js/bookClass.js

```diff
@@ -26,11 +27,9 @@ class Book {
       if (authorVal.trim() === '') return;
       this.bookData.push({ titleVal, authorVal });
       localStorage.setItem('book', JSON.stringify(this.bookData));
-      this.ul.innerHTML += `<li>
-                   <p>${titleVal}</p>
-                   <p>${authorVal}</p>
-                   <button class="remove" value=${titleVal} >Remove</button>
-                   <hr/>
+      this.ul.innerHTML += `<li class="book-item">
+                   <p class="book-data">"${titleVal}" by ${authorVal}</p>
+                   <button class="remove">Remove</button>
                    </li>`;
       this.form.reset();
     });
```

Comment:
- [ ] I'm unable to see the book after adding and in the console, it's giving me an error which I have given screenshot below. Please fix that.

![Screenshot from 2021-11-23 21-42-49](https://user-images.githubusercontent.com/22434660/143061734-1589d7c2-9970-4607-aa2a-fe208bd1b0d3.png)

Reply:
This issue is addressed well. Nice job! :clap: